docs/general/ad-filtering/create-own-filters.md

```diff
@@ -935,6 +948,12 @@ Forbids adding of javascript code to the page. You can read about scriptlets and
 
 - `@@||example.com^$jsinject` disables javascript on pages at `example.com` and all subdomains.
 
+:::caution Restrictions
+
+The `$jsinject` modifier is supported, but not implemented in AdGuard Browser Extension MV3.
```

Comment:
@105th what is this supposed to mean? It cannot be supported and not implemented, it makes literally zero sense.

Reply:
these are not convertible to DNR, we use it only in our tsurlfilter engine to disable some cosmetic rules.